webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/facts/nodes/NodeFactServiceProxies.scala

```diff
@@ -177,6 +178,10 @@ class MockNodeFactFullInventoryRepositoryProxy(backend: NodeFactRepository)
     backend.updateInventory(serverAndMachine, None)(ChangeContext.newForRudder()).unit
   }
 
+  override def saveUserSetAttributes(node: NodeInventory): IOResult[Unit] = {
+    backend.updateInventory(FullInventory(node, None), None)(ChangeContext.newForRudder()).unit
+  }
```

Comment:
this implem is not really used, the inventory tests use the in-memory LDAP backend